Signal/src/ViewControllers/MessageMetadataViewController.swift

```diff
@@ -283,8 +293,7 @@ class MessageMetadataViewController: OWSViewController {
                 bodyLabel.textColor = isIncoming ? UIColor.black : UIColor.white
                 bodyLabel.font = UIFont.ows_regularFont(withSize:16)
                 bodyLabel.text = messageBody
-                // Only show the first N lines.
-                bodyLabel.numberOfLines = 10
+                bodyLabel.numberOfLines = 0
```

Comment:
Great. Is there a vertical scroll indicator?

Also, let's make sure the test plan includes a "send/receive moby dick" case.

Reply:
> Great. Is there a vertical scroll indicator?

Yup - by default

> Also, let's make sure the test plan includes a "send/receive moby dick" case.
 Write

:+1: